portality/bll/doaj.py

```diff
@@ -0,0 +1,620 @@
+from portality import constants
```

Comment:
TODO for me: next review function content more carefully, having left some thoughts on the design.